scss/variables/inputs.js

```diff
@@ -0,0 +1,124 @@
+( function( root, factory ) {
+	if ( typeof define === "function" && define.amd ) {
+		define( [ "./chassis" ], factory );
+	} else if ( typeof exports === "object" ) {
+		module.exports = factory( require( "./chassis" ) );
+	} else {
+		root.chassis = factory( root.chassis );
+	}
+}( this, function( chassis ) {
+
+chassis.inputs = {
+	"shadows": {
+		name: "Styles for input shadows",
+		value: {
+			"disabled": "0 0 2px 0 rgba(0, 0, 0, 0.12)",
+			"normal": "0 2px 2px 0 rgba(0, 0, 0, 0.12)",
+			"focus-base": "0 0 8px 0 rgba(0, 0, 0, 0.08)",
+			"focus-spread": "0 8px 8px 0 rgba(0, 0, 0, 0.18)",
+			"error-base": "0 0 2px 0 rgba(198,62,54,0.24)",
+			"error-spread": "0 2px 2px 0 rgba(198,62,54,0.48)"
+		}
+	},
+	"transitions": {
+		name: "Transition animations",
+		value: {
+			"focus-shadow": "box-shadow .25s",
+			"focus-border-color": "border-color .25s",
+			"blur-shadow": "box-shadow .1s",
+			"blur-border-color": "border-color .1s"
+		}
+	},
+	"label": {
+		name: "Styles for input label",
+		value: {
+			"display": "block",
+			"max-width": "100%",
+			"font-size": "18px",
```

Comment:
These labels are very small compared to inputs. This would look odd if we have an inline input option